Cargo.toml

```diff
@@ -27,20 +24,11 @@ ipnetwork = "0.12"
 pnet_base = { path = "pnet_base" }
 pnet_sys = { path = "pnet_sys" }
 pnet_datalink = { path = "pnet_datalink" }
-pnet_macros = { path = "pnet_macros", optional = true, version = ">=0.9" }
-pnet_macros_support = { path = "pnet_macros_support", version = ">=0.1" }
-pnet_macros_plugin = { path = "pnet_macros_plugin", optional = true, version = ">=0.1" }
+pnet_transport = { path = "pnet_transport" }
+pnet_packet = { path = "pnet_packet" }
 
-clippy = { version = ">=0.0", optional = true }
-syntex = { version = "0.42", optional = true }
 pcap = { version = "0.6", optional = true }
 netmap_sys = { version = ">=0.0", optional = true, features = ["netmap_with_libs"] }
```

Comment:
I realize that `netmap_sys` should also be moved to `pnet_datalink` and not be a dependency of `pnet` directly. But we should keep the feature here: `netmap = ["pnet_datalink/netmap"]` just like the other PR fixing pcap.

Reply:
I'll fix this.

It would be nice if cargo detected these kinds of things.

Reply:
Sure would be. But that would force cargo to actually inspect your code to see what you use. Which would force cargo to have a Rust source code parser. I don't think it does that. Maybe clippy can detect it?